### PR TITLE
Change default Clickable type; minor checkbox fix

### PR DESCRIPTION
**Pull request by dzucconi**
@pepopowitz good idea re: Clickable; for some reason I thought the default button type was `button` but it's actually `submit`

We've never used a Clickable as a submit button before in Force so this shouldn't have any effect/I don't feel like we need to cut a breaking change... but let me know if you think otherwise.

I can't capture this in a test because Enzyme doesn't support event propagation; so just a story for now.

Also: I forgot to prevent the default of spacebar which is to scroll the page. Publishing a canary to verify in force.

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @artsy/palette@13.33.2-canary.858.15065.0
  # or 
  yarn add @artsy/palette@13.33.2-canary.858.15065.0
  ```
</details>

### CODE REVIEW

**Comment by dzucconi** [comment]
I guess I could also just switch back to using Clickable in the checkbox 😅

**Comment by dzucconi** [comment]
Not sure why a canary wasn't published. I'm having issues with yalc/yarn link sooooo 🤔

**Review by pepopowitz — APPROVED** [review]
Nice! 

I agree about a patch version being adequate.

**Comment by damassi** [comment]
> I'm having issues with yalc/yarn link

This force workflow has never failed me:

```bash
cd palette/packages/palette
yarn link
yarn watch

# new pane 
cd force 
yarn link @artsy/palette  
yarn start
```

File: packages/palette/src/elements/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -49,7 +49,8 @@ export const Checkbox: React.FC<CheckboxProps> = ({
   }
 
   const handleKeyPress = (event: React.KeyboardEvent<HTMLDivElement>) => {
-    if (event.code === "Space" && isSelectable) {
+    if (event.key === " " && isSelectable) {
```
Comment (dzucconi) [inline packages/palette/src/elements/Checkbox/Checkbox.tsx:52]: Weird thing: in Force event.code is undefined; event.key is fine in this case/more widely supported anyway; but odd.

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette@13.34.1`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4013.34.1) :rocket:

**Pull request merged**